package.json

```diff
@@ -21,8 +21,9 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@docusaurus/utils": "2.0.0-beta.17",
-    "@docusaurus/utils-validation": "2.0.0-beta.17",
+    "@docusaurus/utils": "~2.0.0-beta.17",
+    "@docusaurus/utils-validation": "~2.0.0-beta.17",
+    "algoliasearch": "^4.13.0",
```

Comment:
@romdej `algoliasearch` is actually not a required dependency. It's probably set as a peer dependency in another dependency, when it should really be an optional peer dependency...